src/main/kotlin/net/irisshaders/lilybot/extensions/moderation/Report.kt

```diff
@@ -83,10 +138,12 @@ class Report : Extension() {
 		user: UserBehavior,
 		messageLog: GuildMessageChannelBehavior,
 		messageAuthor: Member?,
-		reportedMessage: Message
+		reportedMessage: Message,
+		moderatorRole: String?,
+		modActionLog: String?
 	) {
 		messageLog.createMessage {
```

Comment:
I don't follow, what do you mean

Reply:
Rather than using createMessage and then createEmbed, use createMessage with the mod ping as the content field and then embed. I haven't been able to test it but it should work.